FALSEHOODS.md

```diff
@@ -135,9 +135,59 @@ surprising how many false assumptions programmers continue to make about them.
 
 1.  **Phone numbers contain only digits**
 
-    In Israel, certain advertising numbers start with a `*`.
+    In Israel, certain advertising numbers start with a `*`. In New Zealand,
+    non-urgent traffic incidents can be reported by calling `*555` from a mobile
+    phone.
 
 1.  **Phone numbers are always written in ASCII**
 
     In Egypt, it is common for phone numbers to be written in native digits.
 
+1.  **Phone numbers have only one prefix (area code or national destination code) at a given time**
```

Comment:
Should we wrap them like you did the other titles, or have too-long titles? Have forgotten if both "work" but we should be consistent. I feel like line-length was ignored for titles because they don't always work otherwise?